verilog/formatting/tree_unwrapper.cc

```diff
@@ -862,6 +862,17 @@ void TreeUnwrapper::SetIndentationsAndCreatePartitions(
       VisitIndentedSection(node, 0, PartitionPolicyEnum::kFitOnLineElseExpand);
       break;
     }
+    case NodeEnum::kDataDeclaration: {
+      if ((GetParamListFromDataDeclaration(node) ||
+           !SearchSyntaxTree(node, NodekPortActualList()).empty()) &&
+          style_.always_wrap_module_instantiations) {
+        VisitIndentedSection(node, 0, PartitionPolicyEnum::kAlwaysExpand);
+      } else {
+        VisitIndentedSection(node, 0,
+                             PartitionPolicyEnum::kFitOnLineElseExpand);
+      }
```

Comment:
```suggestion
      const auto policy =
          (style_.always_wrap_module_instantiations &&
           (GetParamListFromDataDeclaration(node) ||
            !SearchSyntaxTree(node, NodekPortActualList()).empty()))
              ? PartitionPolicyEnum::kAlwaysExpand
              : PartitionPolicyEnum::kFitOnLineElseExpand;
      VisitIndentedSection(node, 0, policy);
```

In conditions containing logic operators it is generally preferable to put
simple/lightweight subconditions (such as checking a bool variable) before
complex ones (such as searching). Short-circuiting will skip evaluation of
second subcondition when the result of logic operation is already known from
first subcondition.